nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -28,7 +28,7 @@ type Path =
 export const gasRequiredTo: GasRequiredTo = {
   deployInfrastructureContracts: {
     satp: {
-      NitroAdjudicator: 3_577_883, // Singleton
+      NitroAdjudicator: 3_577_871, // Singleton
```

Comment:
Interesting that running prettier would reduce gas consumption 🤔 ...

Reply:
I believe this is explained by two facts:

1. That the bytecode deployed to chain includes a hash of the contract metadata, which in turn depends on the hash of the source code file itself (including filename and path, and presumably without any "normalization" being performed). So changing the source code in anyway, including semantically unimportant changes, may change the hash and therefore _the amount of nonzero bytes_ deployed to chain. 
2. The gas payable depends (in part) on the amount of nonzero bytes sent in the transaction.

Reply:
Sources:

https://docs.soliditylang.org/en/v0.8.13/metadata.html#encoding-of-the-metadata-hash-in-the-bytecode

https://ethereum.github.io/yellowpaper/paper.pdf